File: 867_Transpose_Matrix/code.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& A) {
        vector<vector<int>> transA(A[0].size(), vector<int>(A.size()));
        
        for(int i = 0; i < A.size(); ++i) {
            for (int j = 0; j < A[0].size(); ++j) {
                transA[j][i] = A[i][j];
            }
        }
        
        return transA;
    }
};

File: 867_Transpose_Matrix/code.py
class Solution:
    def transpose(self, A):
        """
        :type A: List[List[int]]
        :rtype: List[List[int]]
        """
        res = []
        
        for i in range(len(A[0])):
            res.append([row[i] for row in A])
                
        return res